fdbserver/ptxn/test/FakeTLog.actor.cpp

```diff
@@ -160,7 +160,7 @@ ACTOR Future<Void> fakeTLogPeek(TLogPeekRequest request, std::shared_ptr<FakeTLo
 	printTiming << "Commit record version range [" << versionRange.first << ", " << versionRange.second << ")"
 	            << std::endl;
 	printTiming << "Received version range [" << receivedVersionRange.first << ", " << receivedVersionRange.second
-	            << "]" << std::endl;
+	            << ")" << std::endl;
```

Comment:
why do we want to change here?

Reply:
This is an indicator that the end version range is exclusive, not inclusive. See, e.g.

 https://en.wikipedia.org/wiki/Bracket_(mathematics)

in the interval section it explains the difference between ] and ).